performance_tests/src/aws_encryption_sdk_performance_tests/master_key_providers/pylintrc

```diff
@@ -0,0 +1,45 @@
+[MESSAGE CONTROL]
```

Comment:
Are all of the pylintrc files the same? If so just put one of them in src/ and it should apply to all files recursively

Reply:
I didn't know that it applies recursively to all files. Thanks for pointing this out!